src/wymeditor/editor/image-handler.js

```diff
@@ -0,0 +1,681 @@
+/* jshint maxlen:100 */
+"use strict";
+
+/*
+ * # The Image Handler
+ *
+ * Give it an editor instance and it will make
+ * most of your image resizing dreams come true.
+ *
+ * ## IE8 Shenanigans
+ *
+ * When IE8 is not longer supported,
+ * `rem` could be used for more accurate UI element dimensions
+ *
+ * ## IE9 Shenanigans
+ *
+ * Dragging and dropping of images is disabled.
+ * See the `_isImgDragDropAllowed` function.
+ *
+ * ## IE8-11 Shenanigans
+ *
+ * SVG images are not scaled.
+ * They are cropped. That's right.
+ * And applying style to them does not help,
+ * as well. Ideas are welcome.
```

Comment:
that sucks, wish I knew more about SVG to help.  I know there are weird quirks between img & object and the other ways of embedding svg, but I don't know the specifics